### PR TITLE
grasslands metadata

### PR DESCRIPTION
**Pull request by mhagel**
This is for sharing purposes.

### CODE REVIEW

File: Carbonplus-Grasslands/cavan-compact.json [hunk 1]
```diff
@@ -0,0 +1,376 @@
+{
```
Comment (blushi) [inline Carbonplus-Grasslands/cavan-compact.json]: What about renaming this folder to `CarbonPlus-Grasslands`?

File: Carbonplus-Grasslands/cavan-compact.json [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
+            "@context": {
+                "type": "@type",
+                "@vocab": "https://purl.org/geojson/vocab#",
+                "coordinates": {
+                    "@container": "@list"
+                }
+            }
+        },
+        "schema:legalName": "New Kayarem Pty Limited",
+        "schema:name": "New Kayarem Pty Limited",
```
Comment (blushi) [inline Carbonplus-Grasslands/cavan-compact.json]: this is just a `regen:Organization` so we don't need the `schema:name`
```suggestion
```

Comment (blushi) [reply Carbonplus-Grasslands/cavan-compact.json]: ditto for all the similar data

File: Carbonplus-Grasslands/cavan-compact.json (hunk 1 above)
Comment (mhagel) [reply Carbonplus-Grasslands/cavan-compact.json]: sgtm

File: Carbonplus-Grasslands/cavan-compact.json (hunk 2 above)
Comment (mhagel) [reply Carbonplus-Grasslands/cavan-compact.json]: done

**Pull request closed without merging**